code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm

```diff
@@ -0,0 +1,199 @@
+/datum/eldritch_knowledge/base_void
+	name = "Glimmer of Winter"
+	desc = "Opens up the path of void to you. Allows you to transmute a knife in a sub-zero temperature into a void blade."
+	gain_text = "I feel a shimmer in the air, atmosphere around me gets colder. I feel my body realizing the emptiness of existance. Something's watching me"
+	banned_knowledge = list(/datum/eldritch_knowledge/base_ash,/datum/eldritch_knowledge/base_flesh,/datum/eldritch_knowledge/final/ash_final,/datum/eldritch_knowledge/final/flesh_final,/datum/eldritch_knowledge/base_rust,/datum/eldritch_knowledge/final/rust_final)
+	next_knowledge = list(/datum/eldritch_knowledge/void_grasp)
+	required_atoms = list(/obj/item/kitchen/knife)
+	result_atoms = list(/obj/item/melee/sickly_blade/void)
+	cost = 1
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/base_void/recipe_snowflake_check(list/atoms, loc)
+	. = ..()
+	var/turf/open/turfie = loc
+	if(turfie.GetTemperature() > T0C)
+		return FALSE
+
+/datum/eldritch_knowledge/void_grasp
+	name = "Grasp of Void"
+	desc = "Lowers your victims body temperature by 20 degrees, also temporarily mutes them."
+	gain_text = "I found the cold watcher who observes me. The resonance of cold grows within me. This isn't the end of the mystery."
+	cost = 1
+	route = PATH_VOID
+	next_knowledge = list(/datum/eldritch_knowledge/cold_snap)
+
+/datum/eldritch_knowledge/void_grasp/on_mansus_grasp(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!iscarbon(target))
+		return
+	var/mob/living/carbon/carbon_target = target
+	var/turf/open/turfie = get_turf(carbon_target)
+	turfie.TakeTemperature(-20)
+	carbon_target.adjust_bodytemperature(-40)
+	carbon_target.silent += 4
+	return TRUE
+
+/datum/eldritch_knowledge/void_grasp/on_eldritch_blade(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		var/datum/status_effect/eldritch/E = H.has_status_effect(/datum/status_effect/eldritch/rust) || H.has_status_effect(/datum/status_effect/eldritch/ash) || H.has_status_effect(/datum/status_effect/eldritch/flesh)  || H.has_status_effect(/datum/status_effect/eldritch/void)
+		if(E)
+			E.on_effect()
+			H.silent += 3
+
+/datum/eldritch_knowledge/cold_snap
+	name = "Aristocrat's Way"
+	desc = "Makes you immune to cold temperatures, you can still take damage from lack of pressure."
+	gain_text = "I found a thread of cold breath. It lead me to a strange shrine, all made of crystals. Translucent and white, a depiction of a nobleman stood before me."
+	cost = 1
+	route = PATH_VOID
+	next_knowledge = list(/datum/eldritch_knowledge/void_cloak,/datum/eldritch_knowledge/void_mark,/datum/eldritch_knowledge/armor)
+
+/datum/eldritch_knowledge/cold_snap/on_gain(mob/user)
+	. = ..()
+	ADD_TRAIT(user,TRAIT_RESISTCOLD,MAGIC_TRAIT)
```

Comment:
man, if only there was a TRAIT_NOFIRE/TRAIT_RESISTHEAT counterpart for this in the ash path (outside of the final ascension ability)...

Reply:
since cold damage doesn't come too much, maybe you could also give vacuum immunity here as well?

although, since this is a tier 1 power, that might turn vacuum exposure into a heretic test...

Reply:
that would be too OP for a T1 power